che--typescript/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
I'm not sure someone would mount a .ssh directory as it will override its existing private keys by the workspace private key 3 lines after
`echo "${SSH_PRIVATE_KEY}" > $HOME/.ssh/id_rsa`